runtime/nls/shrc/j9shr.nls

```diff
@@ -6737,3 +6737,17 @@ J9NLS_SHRC_GET_DIR_NO_SPACE.explanation=Cannot allocate native memory.
 J9NLS_SHRC_GET_DIR_NO_SPACE.system_action=The JVM terminates, unless you have specified the nonfatal option with "-Xshareclasses:nonfatal", in which case the JVM continues without using Shared Classes.
 J9NLS_SHRC_GET_DIR_NO_SPACE.user_response=Ensure the process has enough native memory to run.
 # END NON-TRANSLATABLE
+
+J9NLS_SHRC_SHRINIT_HELPTEXT_NO_CLASSPATH_TIMESTAMP_CHECK=Minimize update checking on jar/zip files on classpaths for optimal performance.
+# START NON-TRANSLATABLE
+J9NLS_SHRC_SHRINIT_HELPTEXT_NO_CLASSPATH_TIMESTAMP_CHECK.explanation=NOTAG
+J9NLS_SHRC_SHRINIT_HELPTEXT_NO_CLASSPATH_TIMESTAMP_CHECK.system_action=
+J9NLS_SHRC_SHRINIT_HELPTEXT_NO_CLASSPATH_TIMESTAMP_CHECK.user_response=
+# END NON-TRANSLATABLE
+
+J9NLS_SHRC_SHRINIT_HELPTEXT_CLASSPATH_TIMESTAMP_CHECK=Always do timestamp checking on classpaths.
```

Comment:
I suggest "Check classpath timestamps on every class load".

Reply:
Already updated against the suggestion above.

Reply:
As per Hang's suggestion, this can be modified to "Check non-bootstrap classpath timestamps on every class load"

Reply:
Already updated as suggested.